setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='ixbrl_viewer',
-    version='0.1.6',
+    version='0.1.7',
```

Comment:
FYI: `os.getenv("GIT_TAG")` might be a good option in the future

Reply:
Opened [XMAS-4198](https://jira.atl.workiva.net/browse/XMAS-4198).